lib/evm/machine_state.ex

```diff
@@ -146,4 +146,19 @@ defmodule EVM.MachineState do
     %{machine_state | active_words: max(machine_state.active_words, last_word)}
   end
 
+  @doc """
+  Pops n values off the stack
+
+  ## Examples
+
+      iex> EVM.MachineState.pop_n(%EVM.MachineState{stack: [1, 2, 3]}, 2)
+      {[1 ,2], %EVM.MachineState{stack: [3]}}
+  """
+  @spec pop_n(MachineState.t, pc) :: {MachineState.t, list()}
```

Comment:
returning `list(EVM.val)` instead of `list()` could probably help the type checker (`EVM.val` is presumed to be wrapped to max EVM int size)